README.md

```diff
@@ -64,9 +64,12 @@ Once you have verified the generated doc on the draft site, open a PR from `draf
+10. Before you proceed, make sure all of the pull requests and changes from previous steps have been merged into their respective repos (docs, docs-generated, and docs-playbook).
+11. Confirm the updated content and release versions on the draft and staging sites.
+   a. The PRs that you merged in step 7 & 8 triggered builds of the [`docs-draft site`](https://docs-draft-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/docs/latest/overview.html), [`full draft site`](https://draft-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/docs/latest/overview.html), [`docs-staging site`](https://docs-staging-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/docs/latest/overview.html), and [`full staging site`](https://staging-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/docs/latest/overview.html). Since the changes made in the previous steps were only to the doc specific repos, you can use the doc specific sites to validate your changes which take ~25min to redeploy (whereas the full sites take ~60min). If you have access, you can track the build progress in the [#was-ol-draft-site-alerts](https://ibm-cloud.slack.com/archives/C01G7L68KAP) and [#was-ol-staging-site-alerts](https://ibm-cloud.slack.com/archives/C01GX9P8YP2) slack channels.
+   b.  Once the builds complete, you can verify them on the draft and staging sites, which should now default to the _next_ upcoming version as `latest`. Look for the next upcoming version in the version-picker display at the beginning of the table of contents. When the prod site publishes, it will default to the version that is being released.
```

Comment:
```suggestion
   b. Once the builds complete, you can verify them on the draft and staging sites, which should now default to the _next_ upcoming version as `latest`. Look for the next upcoming version in the version-picker display at the beginning of the table of contents. When the prod site publishes, it will default to the version that is being released.
```